src/controllers/queries/__tests__/index.test.ts

```diff
@@ -526,8 +575,15 @@ describe('QueriesController', () => {
             status: 'forwarded',
             response: {
               mass: 10,
+              unit: 'kg',
               partialResponses: [],
-              subjectId: '00000000-0000-0000-0000-d8ae0805059e',
+              subjectId: {
+                idType: 'product_and_quantity',
+                content: {
+                  productId: '00000000-0000-0000-0000-d8ae0805059e',
+                  quantity: 2,
+                },
+              },
```

Comment:
Need to be clear in documentation when it's updated that the response value (here 10kg) means the total for the subject (in this case it is the cumulative total for two objects (quantity 2)).

Reply:
Added comment to VR-297